invenio_app_ils/ui/backoffice/src/pages/LoanDetails/state/types.js

```diff
@@ -1,6 +1,6 @@
-export const IS_LOADING = 'IS_LOADING';
-export const IS_ACTION_LOADING = 'IS_ACTION_LOADING';
-export const LOAN_ACTION_SUCCESS = 'LOAN_ACTION_SUCCESS';
-export const LOAN_DETAILS_SUCCESS = 'LOAN_DETAILS_SUCCESS';
-export const LOAN_ACTION_HAS_ERROR = 'LOAN_ACTION_HAS_ERROR';
-export const LOAN_DETAILS_HAS_ERROR = 'LOAN_DETAILS_HAS_ERROR';
+export const IS_LOADING = 'loan details/IS_LOADING';
```

Comment:
Minor suggestion, we could define a prefix for each set of types. Also we don't have to introduce `/` we can use yet another `_`

```
prefix = `LoanDetails`
export const IS_LOADING = `${prefix}_IS_LOADING`;
```

Reply:
Yes I can do that, I guess it is a matter of taste. I honestly like this human readable version:
![screenshot 2018-11-20 at 17 12 27](https://user-images.githubusercontent.com/2089455/48786625-818e7e80-ece7-11e8-949b-da8a4e34237e.png)